### ISSUE TITLE
/checkmoney gets an unknown amount

### ISSUE THREAD

**Issue by Nerahikada** [SOF3/Capital#49]
If you run the `/checkmoney` command immediately after installing the plugin, an unknown value will be displayed.

**Comment by SOF3** [comment]
yes, because the player doesn't have an account yet. you have to spend or get money the first time to actually initialize the account.

**Comment by Nerahikada** [comment]
Why is the account not initialized when a player joins?

**Comment by SOF3** [comment]
> Why is the account not initialized when a player joins?

because players may have multiple accounts. For example, if the world schema (#10) is used, the player accounts do not exist until they join the world.

perhaps it would be useful to try to trigger account initialization regularly to avoid confusion indeed.
